succeeding/notifications/README.md

```diff
@@ -44,255 +44,305 @@ In this step, we use Cargo to create a new project and list the dependencies. We
+    }
+    ```
+
+2. In the `lib.rs` file, create an implementation of the struct `Recipient`.
+
+    ```rust
+    impl Recipient {
 
     }
     ```
 
-2. Convert the Application Instance and Message ID to a Link to listen to.
+3. In the implementation create a Recipient starting from a seed chosen from him and by connecting to the IOTA test network through a public node .
 
     ```rust
-        let announcement_link = Address::from_str(&application_instance,&announcement_id)
-            .expect("Failed to create the Announcement Link");
+        pub fn new(seed: &String) -> Self {
+            Self {
+                inner: Subscriber::new(
+                    seed,
+                    "utf-8",
+                    PAYLOAD_BYTES,
+                    false,
+                    Client::new_from_url("https://api.lb-0.testnet.chrysalis2.com"),
+                )
+            }
+        }
     ```
 
-2. Listen to the Channel.
+4. Create a function that listen to a Channel using the Application Instance and Message ID shared by the Publisher.
 
     ```rust
-        recipient
-            .receive_announcement(&announcement_link)
-            .await
-            .expect("Failed to listen to the Channel");
+        /// Listen to a Channel using the Application Instance and Message ID
+        /// shared by the Publisher.
+        pub async fn listen(
+            &mut self,
+            application_instance: &String,
+            announcement_id: &String,
+        ) {
 
-        println!("Listening to the Channel");
+        }
     ```
 
+5. Convert the Application Instance and Message ID to a Link to listen to.
 
-## Step 6. Receive a notification
-In this step, we write a function that receives a notification from the Channel. The message will be verified and converted from bytes to plain text.
+    ```rust
+            let announcement_link = Address::from_str(&application_instance,&announcement_id)
+                .expect("Failed to create the Link");
+    ```
 
-1. Create a function called `receive`.
+6. Listen to the Channel.
 
     ```rust
-    /// Receive any notifications sent by the Author to the Channel.
-    pub async fn receive<T: Transport>(recipient: &mut Subscriber<T>) {
+            self.inner
+                .receive_announcement(&announcement_link)
+                .await
+                .expect("Failed to listen to the Channel");
+    ```
 
-    }
+
+## Step 6. Receive notifications
+In this step, we write a function that receives notifications from the Channel. The messages will be verified and converted from bytes to plain text.
+
+1. Still in the implementation of the Recipient create a function called `receive`.
+
+    ```rust
+        /// Receive any notifications sent by the Publisher to the Channel.
+        pub async fn receive(&mut self) {
+            
+        }
     ```
 
-2. Fetch any messages pending for our Recipient.
+
+2. Fetch continuosly if there are any messages pending for our Recipient.
```

Comment:
It is not continuously fetching messages. It is one request to the IOTA Client to query new messages on the Tangle, which may return zero or more messages.

```suggestion
2. Fetch any pending messages for our Recipient.
```